File: 프로그래머스/3/12987. 숫자 게임/숫자 게임.py
def solution(A, B):
    A.sort()
    B.sort()

    a_pointer = 0
    b_pointer = 0
    score = 0

    while b_pointer < len(B):
        if B[b_pointer] > A[a_pointer]:
            score += 1
            a_pointer += 1
        b_pointer += 1

    return score

File: 프로그래머스/3/12987. 숫자 게임/숫자 게임.cpp
#include <string>
#include <vector>
#include <algorithm>
using namespace std;

int solution(vector<int> A, vector<int> B) {
    int answer = 0;
    sort(A.begin(),A.end());
    sort(B.begin(),B.end());
    int A_idx=0;
    for(int i=0;i<B.size();i++){
        if(B[i]>A[A_idx]){
            answer++;
            A_idx++;
        }
    }
    return answer;
}